flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/observer/BaseObserver.java

```diff
@@ -63,7 +63,7 @@ protected void observeJmDeployment(
         JobManagerDeploymentStatus previousJmStatus =
                 deploymentStatus.getJobManagerDeploymentStatus();
 
-        logger.info("Observing JobManager deployment status");
+        logger.info("Observing JobManager deployment with status: {}", previousJmStatus.name());
```

Comment:
we could probaby explicitly say `previous` here:
```
Observing JobManager deployment. Previous status: {}
```

Reply:
That is more accurate. Updated.